### PR TITLE
[#42] Image representation fix

### LINKED ISSUE #42: #26 fix
implementation of #26 needs fixes

### PR DESCRIPTION
**Pull request by annaskulimowska**
#42

### CODE REVIEW

File: imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Color.java [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 public final class Color {
-    public static final int minColorValue = 0;
-    public static final int maxColorValue = 255;
+    public static final int MIN_COLOR_CHANNEL_VALUE = 0;
+    public static final int MAX_COLOR_CHANNEL_VALUE = 255;
 
-    public static final Color white = new Color(255, 255, 255);
-    public static final Color black = new Color(0, 0, 0);
+    public static final Color WHITE = new Color(255, 255, 255);
+    public static final Color BLACK = new Color(0, 0, 0);
```
Comment (dkrawiec) [inline imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Color.java:8]: Why not to use MAX_COLOR_CHANNEL_VALUE  and MIN_COLOR_CHANNEL_VALUE ?

File: imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Image.java [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+package pl.edu.uj.kimage.plugin.model;
 
 public final class Image {
     private final Color[] data;
     private final int width;
     private final int height;
 
     public Image(final int width, final int height, final Color[] data) {
+        if (data == null) {
+            throw new IllegalArgumentException("Color data cannot be null");
```
Comment (dkrawiec) [inline imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Image.java]: Well we have a method for this Objects.requireNonNull(T obj, String message)

**Comment by dkrawiec** [comment]
No serious changes, code looks fine.

File: imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Color.java (hunk 1 above)
Comment (annaskulimowska) [inline imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Color.java:8]: beacuse it's already a const

File: imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Image.java (hunk 2 above)
Comment (annaskulimowska) [inline imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/model/Image.java]: good point

**Pull request merged**